daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/TransactionVersion.scala

```diff
@@ -66,6 +66,7 @@ object TransactionVersion {
       v1_14 -> V14,
       v1_15 -> V15,
       v1_dev -> VDev,
+      v2_dev -> VDev,
```

Comment:
oy! I didn't know about `TransactionVersion`. Since it's also backed by proto, we might be able to rename the existing `string version = 1;` to `minor` and add a `major` field with `[default = 1]`

Reply:
I learned about from Remy and he suggested to not change the Transaction proto yet and to map v2dev to dev for now. Then when we're done forking LF, we can have a go a forking transactions but that's apparently going to be much harder because Canton peeks into those.

Reply:
Indeed let's not touch the transaction version for now.